.circleci/config.yml

```diff
@@ -54,11 +54,19 @@ jobs:
       - attach_workspace:
           at: .
       - run:
-          name: Lint code
```

Comment:
Haha me neither, but it was like that before and it didn't want to break it. Might be because we don't want to lint any other folder except udata_hydra and tests - but when looking at the project it seems it would be fine.
So let me change that indeed